funcs.mk

```
@@ -12,6 +13,8 @@ check-$(1): $(1)-autogen.cddl
 
 $(1)-autogen.cddl: $(2)
 	for f in $$^ ; do ( grep -v '^;' $$$$f ; echo ) ; done > $$@
+	$(cddlc) -2tcddl $$@ --start=$(4) > $$@x
+	mv $$@x $$@
```

This can be simplified as:

```suggestion
	$(cddlc) -2tcddl $$^ --start=$(4) > $$@
```